Spelling

jsoref opened the pull request:
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/keptn/commit/54219f3f3ebabaaadbb15b472ad721b2320790ca#commitcomment-47880997

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/keptn/commit/092ed62f51ca35af844048307d3eb8a6e495ffa3

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

jsoref commented on cli/cmd/create_service.go:
```diff
@@ -40,7 +40,7 @@ var crServiceCmd = &cobra.Command{
 			return errors.New("required argument SERVICENAME not set")
 		}
 
-		if !keptncommon.ValididateUnixDirectoryName(args[0]) {
```
This comes from https://github.com/keptn/go-utils/blob/0c7d901bcee968fbdc9f471fb66d7fb265571432/pkg/lib/keptn/keptn_base.go#L222 so it can't be merged as is.

It's possible this is a "public" function that can never be fixed. But perhaps it can be.

I will drop this from this PR as it's too complicated for a generic spelling fix.

jsoref replied:
I've moved this commit to here: https://github.com/jsoref/keptn/commit/spelling-validate

jsoref replied:
This was fixed in #3324

jsoref commented on cli/pkg/platform/platform_openshift.go:
```diff
@@ -152,6 +152,6 @@ func (p openShiftPlatform) authenticateAtCluster() (bool, error) {
 func (p openShiftPlatform) printCreds() {
-	fmt.Println("Openshift Server URL: " + p.creds.OpenshiftURL)
-	fmt.Println("Openshift User: " + p.creds.OpenshiftUser)
+	fmt.Println("OpenShift Server URL: " + p.creds.OpenShiftURL)
+	fmt.Println("OpenShift User: " + p.creds.OpenShiftUser)
```
While I'm correcting the branding for `OpenShift`, I try to skip field names because reasoning through them is too hard. (I missed this one, I'll push a fix shortly.)

jsoref commented on helm-service/controller/onboarder.go:
```diff
@@ -52,8 +52,141 @@ func NewOnboarder(
 	}
 }
 
+<<<<<<< HEAD
```
Sorry

christian-kreuzberger-dtx commented:
Hi and thanks for your PR and informing us about the GH action, I love both!

Unfortunately we are in the middle of preparing a release and your PR would touch quite a lot of files (and as you can see there are already quite some merge conflicts). To avoid some trouble, is it possible to re-do the PR around March 11th? Also, if you'd like, create a PR with the GH action in it, so we can auto-check our PRs!

jsoref commented:
Can you ping me around then? I mostly respond to events.

christian-kreuzberger-dtx commented:
pinging @jsoref - we have released and are willing to accept your PR. Not sure how you handle the conflicts (or can you redo the PR?), please let us know if you need help.

jsoref commented:
I'll rebase shortly

jsoref commented:
@christian-kreuzberger-dtx: updated.

Fwiw, I start by rebasing and resolving, and then I iterate to get the spell checker happy again.

christian-kreuzberger-dtx commented:
CLI build is broken

jsoref commented:
@christian-kreuzberger-dtx: fixed, I had been skipping the `.github/` directory tree (to skip my own metadata), but there's content in the workflows which cared (and was misspelled), so I've adjusted the exclusion to only skip my metadata.

christian-kreuzberger-dtx commented on bridge/client/app/_models/event-icons.ts:
```diff
@@ -1,7 +1,7 @@
   "artifact-delivery": "duplicate",
   "deployment": "deploy",
-  "test": "perfromance-health",
+  "test": "performance-health",
```
Unfortunately that's a typo in another library :D I'll revert this back after the merge, don't worry.
https://barista.dynatrace.com/resources/icons/perfromance-health

jsoref replied:
@lukasholzer the only places I can find this token are in Dynatrace/sketchmine: https://github.com/Dynatrace/sketchmine/search?q=perfromance-health
but that repository is archived.

christian-kreuzberger-dtx reviewed (approved):
LGTM

christian-kreuzberger-dtx commented:
Thanks a lot @jsoref 
> Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

Can you create a PR for this action (and configuration if needed) too?
